### ISSUE TITLE
Grace period in heartbeat (push)

### ISSUE THREAD

**Issue by m-primo** [louislam/uptime-kuma#2445]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API, UI Feature, Other

### 🔖 Feature description

An input for grace period in seconds to wait before putting the monitor in down state.
This would help heartbeat monitoring be better.

### ✔️ Solution

Number input in seconds in Add/Edit monitor page.
The system would wait that period before putting the monitor in down state.

basically the idea of grace period.

**Comment by louislam** [comment]
Please use this, close as there is a similar feature.

![image](https://user-images.githubusercontent.com/1336778/208823297-8a485f1b-86f8-4c64-95e6-17e1fe02b1c1.png)
